packages/core/package.json

```diff
@@ -62,7 +66,6 @@
     "react": "^18.2.0",
     "retry": "^0.13.1",
     "stacktrace-parser": "^0.1.10",
-    "viem": "^1.18.9",
```

Comment:
I'm still wrapping my head around peer dependencies. But, don't we need `viem` as a full dep? The interaction between `@ponder/core` and `viem` is not just in the user repository; it's also in `@ponder/core` itself? Genuinely not sure lol.